core/color/gradation.json

```diff
@@ -1,64 +1,74 @@
 {
   "color": {
     "gradation": {
-      "webex": {
-        "light": {
-          "primary": ["#F0F0F0", "#FFFFFF"],
-          "secondary": ["#FAFAFA", "#FFFFFF"]
-        },
+      "bronze": {
         "dark": {
-          "primary": ["#1A1A1A", "#0F0F0F"],
-          "secondary": ["#262626", "#0F0F0F"]
-        }
-      },
-      "jade": {
-        "light": {
-          "primary": ["#E6F2F1", "#FFFFFF"],
-          "secondary": ["#F7FCFC", "#FFFFFF"]
+          "primary": ["#000000", "#171002"],
+          "secondary": ["#000000", "#171002"]
```

Comment:
Same here, I see 2E2515 instaed of 000000

Reply:
I'm guessing that that is because the design-tokens branch hasn't been updated properly. We should double check these values before merge, but i think once the token assets get updated, this will be resolved (need to run a new publish with design token branch)